constants/trainer_constants.asm

```diff
@@ -1,5 +1,8 @@
+__trainer_class__ = 0
+
 trainerclass: MACRO
-	enum \1
+\1 EQU __trainer_class__
+__trainer_class__ = __trainer_class__ + 1
 const_value = 1
```

Comment:
use const_def?

Reply:
`const_value` is for the individual trainer party constants, `__trainer_class__` is for the class constants. They have to run up in parallel.